ForumProject/projects/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path, include
+from . import views
+from rest_framework import routers
+
+app_name = 'projects'
+
+router = routers.DefaultRouter()
+router.register('', views.ProjectView)
```

Comment:
Make sure that `views.ProjectView` is the correct name of the viewset class you intend to use. Typically, viewsets are named in a way that ends with 'ViewSet' (e.g., `ProjectViewSet`) to distinguish them from regular views

Reply:
Done, thanks